Sources/StreamChatUI/CommonViews/ChatInputView/ChatMessageInputView.swift

```diff
@@ -36,10 +36,19 @@ open class _ChatMessageInputView<ExtraData: ExtraDataTypes>: _View, ComponentsPr
     }
     
     override open func setUpLayout() {
-        embed(container)
+        layoutMargins.left = 8
+        layoutMargins.top = 1
+        layoutMargins.right = 8
+        layoutMargins.bottom = 1
```

Comment:
Maybe add comment why this is needed? :)

 I feel like from documentation of `layoutMargins`, you might be wanting to use `directionalLayoutMargins` instead... Also it says that they have the default value of 8 by default, maybe wanna give it a try? 

https://developer.apple.com/documentation/uikit/uiview/1622566-layoutmargins